src/js/references/components/__tests__/ReferenceList.test.tsx

```diff
@@ -27,13 +27,18 @@ describe("<ReferenceList />", () => {
     afterEach(() => nock.cleanAll());
 
     it("should render correctly", async () => {
+        const permissions = createFakePermissions({ create_ref: true });
+        const account = createFakeAccount({ permissions: permissions });
+        mockAPIGetAccount(account);
```

Comment:
I think adding the account here might be making the: `should not render creation button when [canCreate=true]` test redundant. I would remove the other test unless I am missing something here.

Reply:
Yes, you are right